ntbs-service/Pages/Notifications/Edit/Items/SocialContextBase.cshtml.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ntbs_service.DataAccess;
+using ntbs_service.Helpers;
+using ntbs_service.Models;
+using ntbs_service.Models.Entities;
+using ntbs_service.Services;
+
+namespace ntbs_service.Pages.Notifications.Edit.Items

```

Comment:
Gotta say, I'm always very excited when i get to use generic base classes lol

Reply:
Nice one!